contracts/smart-contract-wallet/SmartAccountFactory.sol

```diff
@@ -101,6 +115,8 @@ contract SmartAccountFactory {
         }
         require(address(proxy) != address(0), "Create call failed");
 
+        bytes memory initializer = getInitializer(_owner);
```

Comment:
Also emit a different event in this function as well.

Reply:
what would that event be?

Reply:
In deployCounterFactualWallet, emit event `CounterFactualAccountCreated`
and in deployWallet emit event `AccountCreated`

Reply:
I see. ok